packages/pyright-internal/package.json

```diff
@@ -29,10 +29,11 @@
         "leven": "3.1.0",
         "source-map-support": "^0.5.21",
         "tmp": "^0.2.1",
-        "vscode-jsonrpc": "^9.0.0-next.4",
-        "vscode-languageserver": "^10.0.0-next.6",
+        "vscode-jsonrpc": "^9.0.0-next.2",
+        "vscode-languageserver": "^10.0.0-next.2",
+        "vscode-languageserver-protocol": "^3.17.6-next.3",
         "vscode-languageserver-textdocument": "1.0.11",
-        "vscode-languageserver-types": "^3.17.6-next.4",
+        "vscode-languageserver-types": "^3.17.6-next.3",
```

Comment:
we seem to be downgrading

Reply:
Yes, that's on purpose. Needed to do this until I can get python extension to upgrade too.